openshift/buildconfig.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  labels:
+    app.kubernetes.io/name: bf2-arch-bot
+  name: bf2-arch-bot
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: bf2-arch-bot:1.0.0-SNAPSHOT
+  runPolicy: Serial
+  source:
+    git:
+      uri: "https://github.com/tombentley/bf2-arch-bot"
```

Comment:
Maybe we should move it under bf2?

Reply:
Completely up to you, but I think it would be a good idea, so that it's discoverable with the rest of the projects (including the architecture project to which it contributes).

Reply:
I created the following if you want to go that direction: https://github.com/bf2fc6cc711aee1a0c2a/org-management/pull/389